### ISSUE TITLE
OS scripts are run as root:gnt-daemons in split user mode

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#763]
Originally reported of Google Code with ID 707.

```
ganeti-noded runs as root:gnt-daemons, so the OS scripts are run with the same GID. As instance-debootstrap is called from ganeti-noded, files created by it have the same GID. This leads to files in the guest having the GID of gnt-daemons.

I don't know if we promise OS scripts to run as root:root. In this case, it's a bug we need to fix in ganeti-noded. Otherwise, we could fix it in instance-debootstrap.

This has been verified for 2.10, but the same code is present in 2.8 and 2.9. So I'll assign the bug to 2.9 as it's the current stable release. The fix might as well be backported to 2.8 if there's the need for it.
```
Originally added on 2014-02-05 13:57:44 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
FYI instance hooks are also affected by this.
```
Originally added on 2014-02-05 16:09:48 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
commit 97b7831a94bad644d580983bf4aee6244bf86ceb
Author: Michele Tartara <mtartara@google.com>
Date:   Wed Feb 5 17:46:32 2014 +0000

    Fix execution group of NodeD
    
    The Node deamon was executed with the wrong gid (gnt-daemons) instead of the
    assigned to it by configure.ac.
    
    Fixes Issue 707.
    
    Signed-off-by: Michele Tartara <mtartara@google.com>
    Reviewed-by: Klaus Aehlig <aehlig@google.com>

```
Originally added on 2014-02-05 17:53:11 +0000 UTC.

Mentioned Issue with GoogleCode ID 707 has been migrated to GitHub with ID #763.
Changed State: Fixed
Added to Milestone: Release2.8
Removed from Milestone: Release2.9

**Comment by Ganeti-Issues-Migrator** [comment]
```
Released in the 2.9.5 released.
```
Originally added on 2014-02-27 11:58:33 +0000 UTC.

Changed State: Released
